recipes/eigen/run_test.bat

```diff
@@ -0,0 +1,3 @@
+cl test.cc
+
+test.exe
```

Comment:
No clue if this is right. Based on my googling in order to reproduce `run_test.sh` for Windows, this is what I came up with, but I feel I need to specify the includes somehow.